### PR TITLE
[recorder] Recorder docs

### PR DESCRIPTION
**Pull request by terryytran**

### CODE REVIEW

**Review by phieutran — APPROVED** [review]
The page's content looks good to me

File: packages/docs/recorderDocs/Actions.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  - **Original coordinates:** `number` The "from" horizontal and vertical coordinate in pixel of the element (not editable).
+  - **Destination coordinates:** `number` The "to" horizontal and vertical coordinate in pixel of the element (editable).
+
+### `DRAG-SELECT`
+
+Drag to select a string of text on the page.
+
+- **Editable:** No
+
+If you'd like to change the selected text, you'll need to drag-select the new text on the page and remove the old action.
```
Comment (HieuLeKMS) [inline packages/docs/recorderDocs/Actions.md:60]: This is not changing the selected text, this is removing the old action and create a new one. And I think `**Editable:** No` above makes it clear enough that you cannot edit the text and have to create a new action
```suggestion
```

File: packages/docs/recorderDocs/RecordScenario.mdx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+Note that while you're editing an action, the Recorder will not record your actions on the page. Once done editing, click `Done` to save the changes or `Cancel` to discard, then continue recording.
+
+For `NAVIGATE` action, you can directly edit the URL on the Recording screen.
+
+### Insert actions
+
+To insert some actions before or after a recorded action:
+
+1. Hover on the action, then click `...` icon > `Move cursor before` or `Move cursor after`. The red line (aka. the cursor) will be moved to the corresponding position.
```
Comment (HieuLeKMS) [inline packages/docs/recorderDocs/RecordScenario.mdx:84]:
```suggestion
1. Hover on the action, then click `...` icon > `Move cursor before` or `Move cursor after`. The red line (aka the cursor) will be moved to the corresponding position.
```

File: packages/docs/sidebarsRecorder.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	recorder: [
+		'getStarted',
+		'record',
+		'assert',
+		'variables',
+		'export',
+		'settings',
+		'actions'
+	],
+}
```
Comment (HieuLeKMS) [inline packages/docs/sidebarsRecorder.js:11]:
```suggestion
}

```

File: packages/docs/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "@tsconfig/docusaurus/tsconfig.json",
+  "include": ["src/"]
+}
```
Comment (HieuLeKMS) [inline packages/docs/tsconfig.json:4]:
```suggestion
}

```

File: packages/docs/recorderDocs/Actions.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
+---
+id: actions
+title: Supported actions
+---
+
+Actions are units that comprise a test scenario. There are two categories of actions:
+
+* **Browser interactions:** actions simulating user interactions and can be recorded by interacting with the browser.
+
+* **Programming actions:** actions to be used for programming purposes, cannot be recorded by interacting with the browser but added from Recorder menus.
```
[the remark this answers is not in the document]
Comment (terryytran) [reply packages/docs/recorderDocs/Actions.md:10]: @nick-flood Could you please weigh in?

[the remark this answers is not in the document]
Comment (terryytran) [reply packages/docs/recorderDocs/Actions.md:10]: These actions can be added from different menus: Add menu, Action context menu, and Variable menu (not just 1 menu). So I think the word choice is fine. Let's wait for @nick-flood to weigh in.

File: packages/docs/recorderDocs/Actions.md [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+
+Click on an element of the page.
+
+- **Editable:** Yes
+
+- **Action type:** Can be changed to `DBCLICK`
+
+- **Action-specic fields:**
+
+  - **Target element:** Element to click on. This can be changed to any visible elements on the page.
```
[the remark this answers is not in the document]
Comment (terryytran) [reply packages/docs/recorderDocs/Actions.md:26]: Actually, action-specific fields are the major reason I added this chapter to the guide. It acts like a "API reference". Each action is sort of mapped with an API, which takes in arguments (action-specific fields). I'm aware that our UI is pretty self-explanatory but just want to be sure we don't miss out on anything that a user may look for.

@nick-flood your thoughts on this, please?

[the remark this answers is not in the document]
Comment (nick-flood) [reply packages/docs/recorderDocs/Actions.md]: I like it, even if its a bit self explanatory, the more the merrier.

File: packages/docs/recorderDocs/Actions.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (nick-flood) [reply packages/docs/recorderDocs/Actions.md:10]: Hmm, @ivanvanderbyl might have something in mind here

File: packages/docs/recorderDocs/Actions.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+
+### `HOVER`
+
+Hover on an element on the page.
+
+To record a hover action, press and hold the right Shift, then hover on the target element and release the right Shift. You can turn this on/off in the [Recorder settings][recorder-settings-hover].
+
+- **Editable:** Yes
+
+- **Action type:** Can not be changed
```
Comment (ivanvanderbyl) [inline packages/docs/recorderDocs/Actions.md:68]: Fix typo

File: packages/docs/recorderDocs/Actions.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+  - **Viewport width:** `number` The width of the viewport in pixel after resizing.
+  - **Viewport height:** `number` The height of the viewport in pixel after resizing.
+
+### `SCROLL`
+
+Scroll the web page up, down, left or right to a coordinate.
+
+- **Editable:** Yes
+
+- **Action type:** Can not be changed
```
Comment (ivanvanderbyl) [inline packages/docs/recorderDocs/Actions.md:111]: Typo

File: packages/docs/recorderDocs/Actions.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+  - **Target element:** The dropdown list to select an item from. This can be changed to any dropdown lists on the page.
+  - **Selected value:** `string` or `number` Value of the item to select.
+
+### `TYPE`
+
+Type a string of text to a textbox or a text area.
+
+- **Editable:** Yes
+
+- **Action type:** Can not be changed
```
Comment (ivanvanderbyl) [inline packages/docs/recorderDocs/Actions.md:137]: Typo

File: packages/docs/recorderDocs/Actions.md [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+
+- **Action-specific fields:**
+
+  - **Target element:** The text area to type into. This can be changed to any text areas on the page.
```
Comment (ivanvanderbyl) [inline packages/docs/recorderDocs/Actions.md:141]: Grammar

File: packages/docs/recorderDocs/AddAssert.mdx [hunk 11]
```diff
@@ -0,0 +1,140 @@
+---
+id: assert
+title: Work with assertions
+---
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+An assertion is a validation step that determines whether the automated test scenario passed or failed, or, in other words, to check if the actual and expected result are equal to each other.
+
+Element Recorder supports different common methods for assertion including:
```
Comment (ivanvanderbyl) [inline packages/docs/recorderDocs/AddAssert.mdx:9]: Grammar

**Review by ivanvanderbyl — CHANGES_REQUESTED** [review]
The docs look to cover all the core concepts. Please correct the typos and grammar before merging. If you don't already have access to Grammarly, please ping me for a license.

**Review by HieuLeKMS — APPROVED** [review]

**Pull request merged**